java/clients/src/main/java/sleeper/status/update/ReinitialiseTable.java

```diff
@@ -149,6 +128,10 @@ void run() throws IOException, StateStoreException {
         }
     }
 
+    protected void initialiseStateStore(TableProperties tableProperties, StateStore stateStore) throws IOException, StateStoreException {
+        InitialiseStateStore.createInitialiseStateStoreFromSplitPoints(tableProperties, stateStore, null).run();
```

Comment:
It might be clearer to use an empty list for the split points rather than null.

Reply:
Done.